.gitignore

```diff
@@ -5,6 +5,9 @@ demo/project_demo00-SecOps/simulator/target/
 demo/project_demo00-SecOps/simulator/Cargo.lock
 demo/pipeline_data
 
+# Used to run earthly locally.
+deploy/.env
```

Comment:
Why is this here?

Reply:
`earthly` doesn't run without this file.  It's used in one of the jobs, but is not in the repo